### PR TITLE
Add meeting notes for 2019-12-05

### PR DESCRIPTION
**Pull request by fotoetienne**
Live notes doc: https://docs.google.com/document/d/1uN1e22_4YEdPw-Gpt52J4fblSU5KmHz3vwqgMz89D8c

Converted using Docs to Markdown version 1.0β17

### CODE REVIEW

File: notes/2019-12-05.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+*   Benjie: I was thinking more like how Open Collective operates, there’s a group the can decide to fund someone to complete a certain task. 
+*   Benjie: It does not need to be just maintainers, it could be open to people in the community
+*   Lee: So Open Collective works more like a bounty program, right?
+*   Benjie: Yeah, I think so.
+*   Lee: One thing I’d like to talk to you (Benjie) about moving funds from the GraphQL foundation.
+*   Benjie: Yeah, that sounds great! I think also just encouraging people to help out on GraphiQL is great. 
+*   Lee: What are some easier ways to contribute to those pools? Maybe just setting up Open Collective or setting up GitHub sponsors. 
+*   Ivan: Set up under Linux Foundation or Kubernetes Foundation or Open Collective and reuse the experience. We need to ask Chris. 
+*   Lee: I know the JS Foundation has something set up too. I will ask them as well. 
+*   [ACTION: Lee] Ask the JS Foundation how they set up funding.
```
Comment (caniszczyk) [inline notes/2019-12-05.md]: we already have funding setup that we have been using with Ivan, LF cover all the fees and anyone can contribute
https://funding.communitybridge.org/projects/graphql

File: notes/2019-12-05.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+    *   “Existing code generated tooling is backwards compatible with Introspection additions”
+*   Next steps?
+
+### Notes
+
+*   Vince: Lots of work in the last week, started doing evaluations of the various criteria and solutions (color-coded system). Our goal was to let anything in and then evaluate it. 
+*   Vince: Added 2 new criteria to push things along, they have to do with backwards compatibility. If we make any changes in the SDL, we want them to be compatible with previous parsing. Same thing with the introspection result. Don’t want to break existing tooling like GraphiQL. 
+*   Vince: They have not been evaluated but I’m hoping people can give their opinions. 
+*   Vince: we've looked at what potential SDL might look like for the various solutions, but we haven't looked at the introspection changes
+*   Lee: This is awesome, dramatic improvement. Especially the crosswise exalution which makes it much easier to read.
```
Comment (sungam3r) [inline notes/2019-12-05.md]: exalution ?

Comment (Alan-Cha) [reply notes/2019-12-05.md]: Sorry! I meant to write "evaluation"!

Comment (sungam3r) [inline notes/2019-12-05.md]:
```suggestion
*   Lee: This is awesome, dramatic improvement. Especially the crosswise evaluation which makes it much easier to read. 
```

**Pull request merged**